diff --git a/dist/index.cjs.js b/dist/index.cjs.js
index 38ebbd413317c26c137e856edd34fee2d79951d3..d9c62124db8d09e2cd049f54c8979f3f2390ba16 100644
--- a/dist/index.cjs.js
+++ b/dist/index.cjs.js
@@ -63,7 +63,7 @@ boolProperties.forEach(function (property) {
 });
 
 core.tryIntrospect();
-function isCodSpeedBenchOptions(options) {
+function isCodSpeedBenchOptions(options = {}) {
   return "uri" in options;
 }
 function withCodSpeed(bench) {
diff --git a/dist/index.es5.js b/dist/index.es5.js
index 5353d0bc6d7ba980e29a14a3c9381a93bec83363..95d44fb46831b88f355e283a7158ff4f5719d2a1 100644
--- a/dist/index.es5.js
+++ b/dist/index.es5.js
@@ -61,7 +61,7 @@ boolProperties.forEach(function (property) {
 });
 
 tryIntrospect();
-function isCodSpeedBenchOptions(options) {
+function isCodSpeedBenchOptions(options = {}) {
   return "uri" in options;
 }
 function withCodSpeed(bench) {
